bin/pycbc_upload_xml_to_gracedb

```diff
@@ -160,10 +163,26 @@ for event in coinc_table:
         r = gracedb.createEvent("CBC", "pycbc", "tmp_coinc_xml_file.xml",
                                 search="AllSky", offline=True).json()
     logging.info("Uploaded event %s.", r["graceid"])
+
+    # upload PSD
     gracedb.writeLog(
         r["graceid"], "PyCBC PSD estimate from the time of event",
         "psd.xml.gz", open("tmp_psd.xml.gz", "rb").read(), "psd").json()
     logging.info("Uploaded file psd.xml.gz to event %s.", r["graceid"])
+
+    # add info for tracking code version
+    gracedb_tag_with_version(gracedb, r['graceid'])
+
+    # document the absolute path to the input file
+    input_file_str = 'Candidate uploaded from ' \
+        + os.path.abspath(args.input_file)
+    gracedb.writeLog(r['graceid'], input_file_str)
+
+    # add the custom log message, if provided
+    if args.log_message is not None:
```

Comment:
is the 'is not None' needed? i.e. is default None?

Reply:
In this case `if args.log_message is not None:` is in practice equivalent to `if args.log_message:`. The default is None if the option is not given. I have a slight preference for the former version, because it asks more explicitly if the option is given or not, rather than asking if `bool(args.log_message)` is True, which depends not only on the option being given or not, but also on what string is being given. For the present case the only difference is noticeable if you pass the silly argument `--log-message ''`: it will create an empty comment with the first version, and not create anything with the second.

Reply:
okay that sounds fair